go/base/tests/test_go_start_conversation.py

```diff
@@ -18,8 +18,6 @@ def setUp(self):
         self.command.stderr = StringIO()
 
     def test_sanity_checks(self):
-        self.assertRaisesRegexp(CommandError, 'provide --email-address',
-            self.command.handle, email_address=None, conversation_key=None)
```

Comment:
This would still be a problem if we used composition instead of inheritance. ;-)